src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -452,7 +452,12 @@ export default (accessToken, userID, opts) => {
     ),
     userByEmailLoader: gravityLoader("user", {}, { method: "GET" }),
     userByIDLoader: gravityLoader((id) => `user/${id}`, {}, { method: "GET" }),
-    userInterestsLoader: gravityLoader("me/user_interests"),
+    meUserInterestsLoader: gravityLoader("me/user_interests"),
```

Comment:
changed the current loader to prefix with `me` (and adjusted the calling code to use this newly named loader).